command/analyzer/initialize/init.go

```diff
@@ -0,0 +1,147 @@
+package initialize
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	validator "github.com/deepsourcelabs/cli/analyzers/validator"
+	"github.com/deepsourcelabs/cli/utils"
+	"github.com/pelletier/go-toml"
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+)
+
+var SDKLanguageOpts = []string{"Python", "Go", "JavaScript"}
+
+type SDKResponse struct {
+	SDKRequired bool
+	SDKLanguage string
+}
+
```

Comment:
```suggestion
type IAnalyzerInitOpts interface {
	SingleLineInputPrompt func(string, string, string) (string, error)
	DescriptionPrompt     func(string, string, string) (string, error)
	ConfirmationPrompt    func(string, string) (bool, error)
	SingleOptionPrompt    func(string, string, []string) (string, error)
}
```

Reply:
You can add this interface and ensure that the struct used in tests satisfy this interface.